packages/cli/src/commands/repo/build.ts

```diff
@@ -179,6 +179,7 @@ export async function command(opts: OptionValues, cmd: Command): Promise<void> {
         await buildBackend({
           targetDir: pkg.dir,
           skipBuildDependencies: true,
+          minify: buildOptions.minify,
```

Comment:
@Rugvip the above line `skipBuildDependencies: true,` prevents other packages being minified when running `yarn run build:all`. Should I set `skipBuildDependencies` to false now that we want to enable minification?

Reply:
As far as I can tell they'll already be minified, because the standard package builds that happen above already forward the minify flag. We wouldn't want to build all deps again in this step, but I think this should already work as expected anyway.